### ISSUE TITLE
Stream can not be started anymore

### ISSUE THREAD

**Issue by MRLB** [maxgalbu/xbmc.plugin.video.nba#103]
Hi,
I am using the latest @ivankokan version. Unfortunately it is not possible anymore to watch live matches. Boston @ Toronto for example does not work.

**Comment by ivankokan** [comment]
Hi @MRLB.

I get the same in my Kodi. The issue is not about streaming as the title suggests but creating menu entries for additional camera feeds.

Disable additional camera feeds in add-on settings until I fix it.

Please provide the log next time.

**Comment by MRLB** [comment]
Thanks. I am not at home an therefore it was not possible for me to add a log file. Do you know the reason? Is there any special letter in the centers feeds menu entries what is not supported at kodi?

**Comment by ivankokan** [comment]
@MRLB No, there was just (still unknown) camera 64. Please test the latest from my fork.

**Comment by ivankokan** [comment]
Done in 2015036.
